CODEOWNERS

```
@@ -93,15 +93,15 @@
 /doc/ref_impl/cntt-ri2/chapters/chapter09.md        @rihabbanday @jgu17 @walterkozlowski
 
 # Reference Conformance 2
-/doc/ref_cert/RC2/chapters/chapter01.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter02.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter03.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter05.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter04.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter06.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter07.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter08.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski 
+/doc/ref_cert/RC2/chapters/chapter01.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter02.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter03.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter05.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter04.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter06.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter07.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter08.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski @steelescotr
```

How about simplifying this like the following:

```suggestion
/doc/ref_cert/RC2/        @walterkozlowski @steelescotr
```